docs/deployment/heroku.md

```diff
@@ -8,6 +8,27 @@ The only thing we'll need for this guide is a working Phoenix application. For t
 
 Our main goal for this guide is to get a Phoenix application running on Heroku.
 
+### Limitations
+
+Heroku is a great platform overall, but for Elixir apps, it comes with a number of limitations you should be aware of. 
```

Comment:
Hm, it doesn't look like there is a distillery guide yet, but there is an exrm guide. I'll take a stab at writing a distillery guide for Phoenix if you like.